### PR TITLE
Added barline and repeat element

### PR DESCRIPTION
**Pull request by profOnno**
This adds barline and repeat elements to musicxml export.

### CODE REVIEW

**Review by guiv42 — COMMENTED** [review]
Thanks for this PR, it works fine :)

Do you want to fix the unused variable problem yourself (and update PR)?
If you want, I can also merge PR as it is now and implement the correction myself.

File: desktop/TuxGuitar-musicxml/src/org/herac/tuxguitar/io/musicxml/MusicXMLWriter.java [hunk 1]
```diff
@@ -166,13 +166,45 @@ private void writeTrack(TGTrack track, Node parent, boolean isTablature){
...
+			this.writeBarline(measureNode, measure);
+			
 			this.writeBeats(measureNode, measure, isTablature);
 			
 			previous = measure;
 		}
 	}
+	
+	private void writeBarline(Node parent, TGMeasure measure) {
+		int repeatClose = measure.getRepeatClose();
```
[the remark this answers is not in the document]
Comment (profOnno) [reply desktop/TuxGuitar-musicxml/src/org/herac/tuxguitar/io/musicxml/MusicXMLWriter.java]: Changed it. Didn't notice the curly line, in Eclipse, telling me the same. I did added the change, did a `git reset --soft`, then a commit, and pushed it. Does that work for this pull request?

File: desktop/TuxGuitar-musicxml/src/org/herac/tuxguitar/io/musicxml/MusicXMLWriter.java [hunk 2]
```diff
@@ -166,13 +166,45 @@ private void writeTrack(TGTrack track, Node parent, boolean isTablature){
...
+		boolean needBarline = measure.isRepeatOpen() || (measure.getRepeatClose() > 0);
+		
+		// TODO: 
+		// add, when available in tuxguitar; alternate repeat, bar-style, coda, wavy-line etc..
```
[the remark this answers is not in the document]
Comment (profOnno) [reply desktop/TuxGuitar-musicxml/src/org/herac/tuxguitar/io/musicxml/MusicXMLWriter.java:182]: I noticed something like that. The comment is somewhat generic. There are parts that are available in `tuxguitar` and can be implemented, and there are parts, not (yet?) implemented in `tuxguitar`. Haven't looked in to the repeat any further. I hope you don't find the comment's to cluttering, the are useful to me.

**Review by guiv42 — APPROVED** [review]

**Comment by guiv42** [comment]
Thanks!

**Pull request merged**